versions.yaml

```diff
@@ -21,8 +21,30 @@
 cloudbuild:
   enable_parallel: true
 versions:
+- dir: 3/debian10/3.8
+  from: marketplace.gcr.io/google/debian10
+  templateSubDir: 3.8
```

Comment:
Do you think it will be changing for subsequent releases?
Maybe better to have old template for 3.7 and common for all 3.8+ ( I mean change the naming so it's not fixed for 3.8 only)

Reply:
changed 3.8 to default